dex-it/src/test/scala/com/wavesplatform/it/MatcherSuiteBase.scala

```diff
@@ -155,7 +156,7 @@ abstract class MatcherSuiteBase
   }
 
   private def print(text: String): Unit = {
-    val formatted = s"---------- $text ----------"
+    val formatted = s"---------- [${LocalDateTime.now(ZoneId.of("UTC"))}] $text ----------"
```

Comment:
Or this is the solution for the question above?

Reply:
Now we have not only a test name in the logs, but the timestamp too